packages/bitcore-node/src/routes/api/block.ts

```diff
@@ -52,6 +53,34 @@ router.get('/:blockId', async function(req: Request, res: Response) {
   }
 });
 
+router.get('at-time/:time', async function(req: Request, res: Response) {
+  let { time, chain, network } = req.params;
+  try {
+    const [block] = await BlockStorage.collection
+      .find({
+        $query: {
+          chain,
+          network,
+          timeNormalized: { $lte: new Date(time) }
+        }
+      })
+      .limit(1)
+      .sort({ timeNormalized: -1 })
+      .toArray();
+
+    if (!block) {
+      return res.status(404).send('block not found');
+    }
+    const tip = await ChainStateProvider.getLocalTip({ chain, network });
+    if (block && tip.height - (<IBlock>block).height > 100) {
```

Comment:
It would be nice that `100` was defined as a constant in `middleware` or something, in case we change it on the future we dont leave inconsistencies around.

Reply:
Added a Confirmations enum